demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -432,7 +455,8 @@ def copy_test_files(self, test_playbooks_dir=TEST_PLAYBOOKS_DIR):
                         new_file_path = self.add_suffix_to_file_path(os.path.join(self.test_bundle,
                                                                                   os.path.basename(new_path)))
                         shutil.copyfile(new_path, new_file_path)
-                        self.add_from_version_to_yml(new_file_path)
+                        if new_file_path.endswith('yml'):
```

Comment:
good safeguard.
did you encounter a file in `TestPlaybook` folder which wasn't a YML file?
if so, was it script related?

Reply:
I am not sure if I encountered any other files, I mostly wrote this to be sure of the functionality of this method.